attribute test fails to compile for g++ 7.3

> terminalpp/test/attribute_test.cpp:32:32: error: converting to ‘std::allocator_arg_t’ from initializer list would use explicit constructor ‘constexpr std::allocator_arg_t::allocator_arg_t()’
     attribute_string { {},  "" },

This can be worked around by changing it for a default constructor.

--- a/test/attribute_test.cpp
+++ b/test/attribute_test.cpp
@@ -29,7 +29,7 @@ TEST_P(attributes_with_strings, can_be_streamed_to_an_ostream)
 
 static attribute_string const attribute_strings[] = {
     // All defaults
-    attribute_string { {},  "" },
+    attribute_string {},
 
     // One non-default value
     attribute_string {

